legalize.js

```diff
@@ -968,8 +976,21 @@
         
         
         /* global publiclyExposedInterface */
+        var _Legalize = window.Legalize;
+
+        function noConflict() {
+            if (window.Legalize !== undefined) {
```

Comment:
if (window.Legalize === publiclyExposedInterface) window.Legalize = _Legalize

this is better, because you only hook up _Legalize if it is the version you introduced.